app/assets/index.html

```diff
@@ -22,6 +22,31 @@
   </head>
 
   <body>
-    <h1>Welcome to Brigade Congress!</h1>
+    <img class='title' src="/images/brigade-congress-2020.png"/>
+    <section class='intro'>
+      <p>Welcome to this year's brigade congress! See below for helpful guides and the other tabs above!</p>
+      <h3>Helpful Guides:</h3>
+      <ul class='guides'>
+        <li><a>Guide to using Qiqochat</a></li>
+        <li><a>Guide to using Zoom</a></li>
+      </ul>
+
+    </section>
+
+    <style>
+      .title {
+        display: block;
+        margin-left: auto;
+        margin-right: auto;
+        width: 50%;
+      }
+      .intro {
+        text-align: center
+      }
+      .guides {
+        text-decoration: none;
+        list-style-type: none;
+      }
```

Comment:
cool, thanks. This should go in app/app.scss, which is included on this page. I can move it there.